src/Microsoft.Data.Sqlite.Core/SqliteValueBinder.cs

```diff
@@ -64,18 +71,45 @@ public virtual void Bind()
             }
             else if (type == typeof(char))
             {
-                var value = (long)(char)_value;
-                BindInt64(value);
+                var chr = (char)_value;
+                if (_sqliteType == SqliteType.Text)
+                {
+                    var value = new string(chr, 1);
```

Comment:
@bricelam - Are all this temp variables are for readability?

Reply:
Copy-paste. I wanted to go through and clean it up a bit (And switch to  C# 7 `case` statements in a few places.)

Reply:
@AlexanderTaeschner - You can leave these as is. We will clean it up later.
R# :trollface: